persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -42,6 +42,7 @@ private PersistToolsConstants() {
     public static final String KEY_HOST = "host";
 
     public static final String CONFIG_SCRIPT_FILE = "Config.toml";
+    public static final String PERSIST_TOML_FILE = "Persist.toml";
```

Comment:
Wasn't this a constant in the precious PR too? Was it added again

Reply:
It was added here since this PR was created before merging the previous PR, I'll remove overlapping items if any

Reply:
Removed in [36fe5f0](https://github.com/ballerina-platform/persist-tools/pull/54/commits/36fe5f0b814d539d7d6bdb0bbf09e3dbc175efc6)